dynamiqs/mesolve/mesolve.py

```diff
@@ -168,18 +169,18 @@ def mesolve(
         )
 
     # format and batch all tensors
-    # H: (b_H, 1, n, n)
-    # rho0: (b_H, b_rho0, n, n)
+    # H: (b_H, 1, 1, n, n)
+    # jump_ops: (1, b_jump_ops, 1, n, n)
```

Comment:
Let's rename `b_jump_ops` in `b_L` for consistency with the rest of the code. I know it's not what's done for the state batching `b_rho0`, we could fix it as well but I propose not to do it in this PR, which is already quite cluttered with comments update.

Reply:
Also in `smesolve`.